person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :name, :age, :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
```

Comment:
The can_use_services? is already public, If we don't use private before def the method will become public.

Reply:
- [ ] Good job changing the attribute setters as the previous CR suggested 👍 however you should address the suggestion of the `private` methods as well, because all methods below the `private` keyword are private. you should either put method `can_use_services?` above the `private` keyword or define it `public` using the `public` keyword.